deploy/bootstrap/lega.sh

```diff
@@ -248,7 +248,7 @@ cat >> ${PRIVATE}/lega.yml <<EOF
     restart: on-failure:3
     networks:
       - lega
-    entrypoint: ["gosu", "lega", "ega-finalize"]
+    entrypoint: ["gosu", "lega", "ega-id-mapper"]
```

Comment:
What's the point of renaming id-mapper to finalize when you have to do a ugly hack to get the tests to pass?

Reply:
The modules are up-to-date, but simply because we didn't re-build the image, the ega-finalize does not exist in the nbisweden/ega-base image (but ega-id-mapper does).

The tests do not cover the ega-finalize/id-mapper.
However, when we call it ega-finalize, it of course crashes. Combine that with the fact that the tests do count the number of components up, when one is down, none of the tests run.

We thought the simplest way was to keep the name ega-id-mapper, rather than updating the tests to avoid this situation. We'll tear them down in the next commit anyway.

Reply:
Marking the conversation as resolved, to move forward on the todo list, as this is not a blocker